.travis/s2n_travis_build.sh

```diff
@@ -87,7 +87,10 @@ fi
 if [[ "$TESTS" == "ALL" || "$TESTS" == "sawHMACFailure" ]]; then make -C tests/saw failure-tests ; fi
 if [[ "$TESTS" == "ALL" || "$TESTS" == "ctverif" ]]; then .travis/run_ctverif.sh "$CTVERIF_INSTALL_DIR" ; fi
 if [[ "$TESTS" == "ALL" || "$TESTS" == "sawSIKE_r1" ]]; then make -C tests/saw sike_r1 ; fi
+if [[ "$TESTS" == "ALL" || "$TESTS" == "sawSIKE_r2" ]]; then make -C tests/saw sike_r2 ; fi
+if [[ "$TESTS" == "ALL" || "$TESTS" == "sawSIKE_r2_x86" ]]; then make -C tests/saw sike_r2_x86 ; fi
```

Comment:
Nit: To maintain consistent naming conventions, I think `_x64` would be preferable to `_x86`.